tests/Integration/Behaviour/Features/Scenario/Product/update_pack.feature

```diff
@@ -198,40 +198,40 @@ Feature: Add product to pack from Back Office (BO)
+      | productSkirt1 | regular skirt - Size - S, Color - White | productSkirt1SWhite | 10       | http://myshop.com/img/p/{no_picture}-small_default.jpg |
+      | productSkirt1 | regular skirt - Size - M, Color - White | productSkirt1MWhite | 11       | http://myshop.com/img/p/{no_picture}-small_default.jpg |
+      | productSkirt1 | regular skirt - Size - M, Color - Black | productSkirt1MBlack | 12       | http://myshop.com/img/p/{no_picture}-small_default.jpg |
+      | product2      | shady sunglasses                        |                     | 2        | http://myshop.com/img/p/{no_picture}-small_default.jpg |
+    And I update pack productPack4 with following product quantities:
       | product       | combination         | quantity |
       | productSkirt1 | productSkirt1SWhite | 10       |
       | productSkirt1 | productSkirt1MBlack | 9        |
       | product2      |                     | 2        |
+    Then pack productPack4 should contain products with following details:
+      | product       | name                                    | combination         | quantity | image url                                              |
+      | productSkirt1 | regular skirt - Size - S, Color - White | productSkirt1SWhite | 10       | http://myshop.com/img/p/{no_picture}-small_default.jpg |
+      | productSkirt1 | regular skirt - Size - M, Color - Black | productSkirt1MBlack | 9        | http://myshop.com/img/p/{no_picture}-small_default.jpg |
+      | product2      | shady sunglasses                        |                     | 2        | http://myshop.com/img/p/{no_picture}-small_default.jpg |
     Then product "productPack4" type should be pack
 
   Scenario: I remove all products from existing pack when it contains combination and standard products
     Given product "productPack4" type should be pack
-    And pack productPack4 should contain products with following quantities:
-      | product       | combination         | quantity |
-      | productSkirt1 | productSkirt1SWhite | 10       |
-      | productSkirt1 | productSkirt1MBlack | 9        |
-      | product2      |                     | 2        |
-    When I remove all products from pack productPack4
+    When pack productPack4 should contain products with following details:
+      | product       | name                                    | combination         | quantity | image url                                              |
+      | productSkirt1 | regular skirt - Size - S, Color - White | productSkirt1SWhite | 10       | http://myshop.com/img/p/{no_picture}-small_default.jpg |
+      | productSkirt1 | regular skirt - Size - M, Color - Black | productSkirt1MBlack | 9        | http://myshop.com/img/p/{no_picture}-small_default.jpg |
+      | product2      | shady sunglasses                        |                     | 2        | http://myshop.com/img/p/{no_picture}-small_default.jpg |
```

Comment:
You need to test at least one example where an image is associated For now all the tested products have no images so only the "no picture" URL has been tested

Reply:
Code change to provide line as :
regular skirt : Size - M, Color - Black
Test also refactored inthsi way

Reply:
Yes I'm not talking about the name column but the image column
In these tests they always use the `no_picture` URL, we need at least one use case where an image is present to ensure the URL is correctly generated with the image ID